### PR TITLE
PullRequest handson

### PR DESCRIPTION
**Pull request by noemiguzo**
Meet Me Halfway -  Black Eyed Peas

### CODE REVIEW

**Comment by noemiguzo** [comment]
@ArielPuma747  please could you review the PR?

File: Noemi-Guzman.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Meet Me Halfway -  Black Eyed Peas
+
+I spent my time just thinking thinking thinking about you
```
Comment (ArielPuma747) [inline Noemi-Guzman.txt]: la palabra correcta es bout

File: Noemi-Guzman.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+Every sinple day yes, I'm really missin' missin' you
```
Comment (ArielPuma747) [inline Noemi-Guzman.txt]: la palabra correcta es single

File: Noemi-Guzman.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+And all those things we use to use to use to doing
```
Comment (ArielPuma747) [inline Noemi-Guzman.txt]: la palabra correcta es do

File: Noemi-Guzman.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+Hi girl, whats up, it use, it use to be just me and you
+I spent my time just thinking thinking thinking 'bout you
+Every single day, No I'm really missing missing you
```
Comment (ArielPuma747) [inline Noemi-Guzman.txt]: la palabra correcta es Yes

**Comment by ArielPuma747** [comment]
@noemiguzo added comments

File: Noemi-Guzman.txt (hunk 1 above)
Comment (noemiguzo) [reply Noemi-Guzman.txt]: done

File: Noemi-Guzman.txt (hunk 2 above)
Comment (noemiguzo) [reply Noemi-Guzman.txt]: done

File: Noemi-Guzman.txt (hunk 3 above)
Comment (noemiguzo) [reply Noemi-Guzman.txt]: done

File: Noemi-Guzman.txt (hunk 4 above)
Comment (noemiguzo) [reply Noemi-Guzman.txt]: done

**Review by ArielPuma747 — APPROVED** [review]

**Pull request merged**